phpunit/1_Unit/ComputerUpdateTest.php

```diff
@@ -1285,4 +1301,23 @@ public function SoftwareUniqueForTwoComputers() {
    }
 
 
+   /**
+    * @test
+    * @depends AddComputer
+    */
+   public function ComputerPowerSupply() {
+      global $DB;
+
+      $DB->connect();
+
+      $a_data = getAllDatasFromTable("glpi_devicepowersupplies");
+
+      $this->assertEquals(1, count($a_data), "Must have 1 kind of power supplies created");
+
+      $a_dataLink = getAllDatasFromTable("glpi_items_devicepowersupplies",
```

Comment:
Will fail on current GLPI master since raw SQL in condition is no longer allowed. use:
```
[
   'itemtype'   => 'Computer',
   'items_id'   => 1
]
```